osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -82,6 +85,8 @@ protected override void LoadComplete()
         }
 
         private SkinnableSound spinningSample;
+        private const float SPINNING_SAMPLE_INITAL_FREQUENCY = 1.0f;
```

Comment:
typo in name
```suggestion
        private const float SPINNING_SAMPLE_INITIAL_FREQUENCY = 1.0f;
```
(+ usages)

Reply:
also see notes from CI (both consts should be lower snake-case)